best-practices.md

```diff
@@ -0,0 +1,55 @@
+# Best practices when creating a workflow
+A workflow should be developed in accordance with the guidelines outlined in the  [serverless workflow definitions][1] documentation.
```

Comment:
nit: "the  [serverless" two whitespace

Reply:
nit: I think it's better to use "Serverless Workflow definition" (capitol letters) as defined in the provided link.

Reply:
Thank you, both fixed.